### ISSUE TITLE
Bug on Test Creation

### ISSUE THREAD

**Issue by Gauntlet173** [Lexpedite/blawx#210]
There is some way to create a test inside a ruledoc and name it in such a way that all of the RuleDoc pages will fail to load. When I did it, the name was capitalized, had spaces in it, and ended with a numeral, but I don't know what caused the problem. The solution was to delete the offending test and all other tests for that ruledoc, and start again.

**Comment by murtaza64** [comment]
I think spaces cause the issue--I created a test with a space in it and returning to the project page resulted in a 500 server error, but creating a new project and not using a space worked fine

**Comment by Gauntlet173** [comment]
Yeah, thanks, @murtaza64 . I suspect the problem is that the test_name is a `slug` in the database, but the input is not being validated for only valid slugs. Really, we ought to be generating slugs from the test name, not directly inputing them.
